build-system/babel-plugins/babel-plugin-transform-internal-version/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Reassigns the trailing comments of a statement to be leading comment of its
+// next sibling. This is because JSDoc comments (which should be on the next
+// statement) get erroneously assigned as trailing comments of this statement.
+module.exports = function (babel, options = {}) {
+  const {types: t} = babel;
+  const {version} = options;
+
+  if (!version) {
+    throw new Error(
+      'Need to specify runtime version as an option to babel transformer'
+    );
+  }
+
+  return {
+    name: 'ast-transform', // not required
```

Comment:
nit: either remove or give real name

Reply:
How about calling this `internal-version`?

Reply:
Done.